operational-mocks.cabal

```diff
@@ -15,8 +15,11 @@ library
   build-depends:
       base
     , operational
+    , free
```

Comment:
This actually ties in quite a few indirect dependencies. Maybe it's better to put this in a separate package. (Also `operational-mocks` as a package name is probably not ideal.) The separate package could be included in this repo.